docs/troubleshooting.md

```diff
@@ -100,3 +101,44 @@ If runs from an Illumina sequencer are universally failing, ensure that
 runscanner-illumina is installed correctly, as outlined in
 [Illumina Setup](../illuminasetup/). Illumina runs
 cannot be read without this interoperability application.
+
+
+## Analysis Output
+
+### Illumina DRAGEN
+
+#### Run with analysis says `analysisExpected` is `false`
+1. Please ensure that the processor configuration has `scanDragen` set to `true`. Please see
+   [Appendix A: Processor Definitions](./appendices.md) for details on configuration.
+1. Run Scanner tests for whether analysis is expected by checking for the presence of an `Analysis`
+directory at the root of the run directory. If no directory is present, please check your sequencer
+configuration. If this directory is present, please check its file permessions, and try
+invalidating the cache for the run via the REST API.
+
+#### Run with analysis only shows some workflows, not all
+When a pipeline run contains a samplesheet which indicates a combination of workflows supported and 
+unsupported by Run Scanner, Run Scanner will parse and serve only the workflows it supports.
+Please see [Illumina DRAGEN](./analysis.md) for the current list of DRAGEN workflows supported
+by Run Scanner.
+
+#### Pipeline run has a `pipelineStatus` of `UNSUPPORTED`
+When a pipeline run contains a samplesheet which indicates only workflows
+unsupported by Run Scanner, or the DRAGEN version indicated in the samplesheet is lower than
+Run Scanner's minimum supported version (4.1.7), Run Scanner sets `pipelineStatus` to `UNSUPPORTED`.
+Please see [Illumina DRAGEN](./analysis.md) for the current list of DRAGEN workflows supported
+by Run Scanner.
+
+#### Pipeline run has a `pipelineStatus` of `SCAN_ERROR`
+A supported workflow has the correct files, however, there is some malformation in the file content
+such that Run Scanner is unable to parse the file. This is an error which requires developer 
+attention. 
+There should be an error message in your tomcat logs that looks like "SCAN_ERROR in [filename] for
+[directory]". If you are able, please
+[open a GitHub Issue](https://github.com/miso-lims/runscanner/issues/new) detailing the situation
+and including the file contents.
+
+#### Workflow run has a status of `FAILED`
+Run Scanner will set a workflow run to `FAILED` if Run Scanner is unable to locate all of the 
+information it requires to serve analysis data, but DRAGEN has indicated via touchfile that 
```

Comment:
This matches the description of `PENDING` status from the "About Workflow Run Status," where `FAILED` is described as "Run Scanner has evidence of a failure of the workflow run."

Reply:
PENDING is "Run Scanner has not found all of the information it needs to serve analysis data yet", FAILED is "Run Scanner is unable to locate all of the information it requires to serve analysis data, **but DRAGEN has indicated via touchfile that it is done writing out analysis data**". The implication is that the missing data can never appear. How would you recommend I clarify this?

Reply:
`FAILED` means Run Scanner thinks the workflow is failed, which seems obvious without explanation, but becomes unclear when reading this.

One reason it might be marked `FAILED` when that is incorrect is that required files or permissions are missing. It may be the only reason for now, but that's likely to change, and there's a lot of documentation here that you're assuming will be accurately maintained with every future change.